src/webextension/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+
+  "description": "WebCompat Go Faster Web Extension",
+  "manifest_version": 2,
+  "name": "webcompat-extension",
+  "version": "1.0",
+
+  "permissions": [
+    "tabs",
+    // TODO: Fill required URL for override rules the same as ua_overrides.js
+    // For tablog.js to have permission to inject script to web console.
+    // Example: "*://www.google.com.tw/*"
```

Comment:
@denschub: yes, all_urls is a special host permission, we can refer to:
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/permissions#Host_permissions
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Match_patterns#%3Call_urls%3E

I think the permission check is on-air, while I want to inject script, ExtensionContent.jsm will check and through exception. All WebExtension use this design, so far I don't see performance issue here.

For manifest.json, it can't be set on runtime. But we can set from ua_overrides.js before we install to gecko-dev by jake.